### DISCUSSION TITLE
Run in system-space as root

### DISCUSSION THREAD

**Discussion by wrein** [abiosoft/colima#974] [General]
Can Colima run in system-space and start at boot time?
something like that:
```bash
sudo brew services start colima
```

**Comment by suhailskhan** [comment]
I too am looking to be able to do something like this. I have a service running in a container that I’d like to keep accessible even after a reboot, without needing to log in first. My container is configured to automatically start with the Docker daemon.

I just tried running `sudo brew services start colima`, and it returned the following output:
```
Warning: Taking root:admin ownership of some colima paths:
  /usr/local/Cellar/colima/0.6.10/bin
  /usr/local/Cellar/colima/0.6.10/bin/colima
  /usr/local/opt/colima
  /usr/local/opt/colima/bin
  /usr/local/var/homebrew/linked/colima
This will require manual removal of these paths using `sudo rm` on
brew upgrade/reinstall/uninstall.
Warning: colima must be run as non-root to start at user login!
==> Successfully started `colima` (label: homebrew.mxcl.colima)
```
I then rebooted and tested to see if the service was still accessible, to no avail.

To my understanding, Colima is not designed to be run as root, which means it can’t be used as a daemon in system-space.

**Reply by suhailskhan** [reply]
@wrein Have you since found a workaround/alternative solution?
